app/src/main/java/com/teamagam/gimelgimel/app/map/view/DrawActionActivity.java

```diff
@@ -0,0 +1,46 @@
+package com.teamagam.gimelgimel.app.map.view;
+
+import android.os.Bundle;
+
+import com.teamagam.gimelgimel.R;
+import com.teamagam.gimelgimel.app.GGApplication;
+import com.teamagam.gimelgimel.app.common.base.view.activity.BaseActivity;
+
+public class DrawActionActivity extends BaseActivity<GGApplication> {
+
+    private BaseDrawActionFragment mMainFragment;
+    private DrawActionBottomPanelFragment mBottomPanelFragment;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        initMainFragment();
+        initBottomPanelFragment();
+    }
+
+    @Override
+    protected int getActivityLayout() {
+        return R.layout.activity_action_draw;
+    }
+
+    private void initMainFragment() {
+        mMainFragment = getMainFragment();
+        getSupportFragmentManager()
+                .beginTransaction()
+                .add(R.id.draw_action_fragment_container, mMainFragment)
+                .commit();
+    }
+
+    private BaseDrawActionFragment getMainFragment() {
+        return new ExampleDrawActionFragment();
```

Comment:
If I understood you yesterday, here in future tasks will be a switch case based on the bundle we talked about?
Or we can make this method protected abstract and implement in extending activities...